roles/pulp_repository/tasks/container.yml

```diff
@@ -16,9 +16,19 @@
     password: "{{ pulp_password }}"
     validate_certs: "{{ pulp_validate_certs | bool }}"
     name: "{{ item.name }}-remote"
-    url: "{{ item.url }}"
-    policy: "{{ item.policy }}"
+    ca_cert: "{{ item.ca_cert | default(omit) }}"
+    client_cert: "{{ item.client_cert | default(omit) }}"
+    client_key: "{{ item.client_key | default(omit) }}"
```

Comment:
This shows up in the ansible output, we should use loop_control/label to prevent that.